tracer/src/Datadog.Tracer.Native/iast/method_info.h

```diff
@@ -56,18 +55,14 @@ namespace iast
         mdTypeDef GetTypeDef();
 
         ModuleInfo* GetModuleInfo();
-        WSTRING GetName();
-        WSTRING GetMemberName();
-        WSTRING GetFullName();
+        WSTRING& GetName();
+        WSTRING& GetFullName();
         WSTRING GetFullNameWithReturnType();
```

Comment:
Why does all other functions returning `WSTRING` changed to `WSTRING&` but not this one?